### PR TITLE
dll output

### PR DESCRIPTION
**Pull request by saiprasadsamudrala**
@maneetgoyal

### CODE REVIEW

**Comment by maneetgoyal** [comment]
@saiprasadsamudrala Resolve the merge conflicts and ping again thereafter

File: src/headers.ts [hunk 1]
```diff
@@ -425,20 +421,20 @@ export const inchi_StereoType0D = new Enum({
...
 export const inchi_StereoParity0D = new Enum({
   INCHI_PARITY_NONE: 0,
-  INCHI_PARITY_ODD: 1, // 'o'
-  INCHI_PARITY_EVEN: 2, // 'e'
-  INCHI_PARITY_UNKNOWN: 3, // 'u'
-  INCHI_PARITY_UNDEFINED: 4, // '?'
+  INCHI_PARITY_ODD: 1 /* 'o' */,
+  INCHI_PARITY_EVEN: 2 /* 'e' */,
+  INCHI_PARITY_UNKNOWN: 3 /* 'u' */ /* (see also readinch.c) used in: Extract0DParities, InchiToAtom  */,
+  INCHI_PARITY_UNDEFINED: 4 /* '?' -- should not be used; however, see Note above */,
```
Comment (maneetgoyal) [inline src/headers.ts:437]: Undo these changes

File: src/headers.ts [hunk 2]
```diff
@@ -404,16 +404,12 @@ export const inchi_Atom = NAPIStructType({
...
-// #define NO_ATOM          (-1) /* non-existent (central) atom */
-
-export const NO_ATOM = -1; // non-existent (central) atom
-
 // /* 0D parity types */
 // typedef enum tagINCHIStereoType0D {
-//     INCHI_StereoType_None = 0,
-//     INCHI_StereoType_DoubleBond = 1,
-//     INCHI_StereoType_Tetrahedral = 2,
-//     INCHI_StereoType_Allene = 3
```
Comment (maneetgoyal) [inline src/headers.ts:416]: Run `yarn pretty` and then commit the changes again

**Review by maneetgoyal — CHANGES_REQUESTED** [review]

File: src/headers.ts [hunk 3]
```diff
@@ -458,6 +454,236 @@ export const inchi_StereoParity0D = new Enum({
...
+//     char *szOptions;    /* InChI options: space-delimited; each is preceded by  */
+//                         /* '/' or '-' depending on OS and compiler */
+// } inchi_InputINCHI;
+
+export const inchi_InputINCHI = NAPIStructType({
+  szInChI: refNAPI.refType(refNAPI.types.char),
+  szOptions: refNAPI.refType(refNAPI.types.char),
+});
+=======
+/*************************************************
```
Comment (maneetgoyal) [inline src/headers.ts:670]: Merge conflict is not properly resolved. The diff should only show the changes that you **added**. It should not undo the code that is already present in the `main` branch

Comment (saiprasadsamudrala) [reply src/headers.ts]: okay sir

**Comment by saiprasadsamudrala** [comment]
Sir, Changes requested are made. If anything is wrong, inform me.

File: src/enum.d.ts [hunk 4]
```diff
@@ -1 +1 @@
-declare module 'enum';
\ No newline at end of file
+declare module "enum";
```
Comment (maneetgoyal) [inline src/enum.d.ts]: Revert it

File: src/headers.ts [hunk 5]
```diff
@@ -406,7 +406,7 @@ export const inchi_Atom = NAPIStructType({
 
 // #define NO_ATOM          (-1) /* non-existent (central) atom */
 
-export const NO_ATOM = -1; // non-existent (central) atom
+export const NO_ATOM = -1; // non-existent (central) atom// /* 0D parity types */
```
Comment (maneetgoyal) [inline src/headers.ts:409]: Revert it

File: src/headers.ts [hunk 6]
```diff
@@ -467,140 +474,203 @@ export const inchi_StereoParity0D = new Enum({
...
 /*
-  Structure -> InChI
+    Structure -> InChI
 
-  GetINCHI()
-  GetStdINCHI()
-  GetINCHIEx()
+    GetINCHI()
+    GetStdINCHI()
+    GetINCHIEx()
```
Comment (maneetgoyal) [inline src/headers.ts:481]: - Why are these changes present?
- Revert them
- Running `yarn pretty` may auto-fix it.
- If that doesn't happen, fix it manually.

File: src/headers.ts [hunk 7]
```diff
@@ -467,140 +474,203 @@ export const inchi_StereoParity0D = new Enum({
...
+//     /* the caller is responsible for the data allocation and deallocation                           */
+
+//     /* same as in older inchi_Input                                                                 */
+//     inchi_Atom *atom;                       /* array of num_atoms elements                          */
+//     /* same as in older inchi_Input                                                                 */
+//     inchi_Stereo0D *stereo0D;               /* array of num_stereo0D 0D stereo elements or NULL     */
+//     /* same as in older inchi_Input                                                                 */
+//     char *szOptions;                        /* InChI options: space-delimited; each is preceded by  */
+//                                             /* '/' or '-' depending on OS and compiler              */
+//     /* same as in older inchi_Input                                                                 */
```
Comment (maneetgoyal) [inline src/headers.ts:633]: Revert all these whitespace changes.

**Review by maneetgoyal — CHANGES_REQUESTED** [review]

File: src/enum.d.ts (hunk 4 above)
Comment (saiprasadsamudrala) [reply src/enum.d.ts]: this fix is done by `yarn pretty`

Comment (maneetgoyal) [reply src/enum.d.ts]: Sync your branch with the `main` branch

**Comment by saiprasadsamudrala** [comment]
Can I close the pull request sir?

**Comment by maneetgoyal** [comment]
Yeah Sai, I have manually moved all the relevant changes from this PR to the `main` branch. Focus on porting remaining entities.
